style.css

```diff
@@ -781,3 +781,153 @@ p {
 .input1 > label {
   display: none;
 }
+
+/* Mobile Popup */
+
+.popUp {
+  position: fixed;
+  top: 0;
+  background-color: white;
+  padding: 24px;
+}
+
+.mobile-pop-up {
+  padding: 16px 16px 20px 16px;
+  border: 2px solid black;
+  margin: 60px 16px 211px 16px;
+  background-color: white;
+  max-width: 1156px;
+}
+
+.pop-up-top {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin-right: 16px;
+  height: 82px;
+}
+
+.pop-up-h1 {
+  width: 80%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 20;
+  font-size: 25px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0;
+  margin-left: 30px;
+  padding-top: 10px;
+}
+
+.close-icon-2 {
+  margin-right: 14px;
+  width: 24px;
+  height: 24px;
+  margin-top: 5px;
+}
+
+.pop-up-languages {
+  list-style-type: none;
+  display: flex;
+  gap: 12px;
+  padding-left: 0;
+  margin-left: 30px;
+}
+
+.pop-up-language-li {
+  border: 1px solid #979493;
+  align-items: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #3c3a39;
+  padding: 12px;
+}
+
+.pop-up-image {
+  width: 80%;
+  height: 300px;
+  padding-left: 30px;
+}
+
+.pop-up-description {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 30px;
+  margin-top: 20px;
+}
+
+.pop-up-buttons {
+  display: flex;
+  margin-top: 12px;
+  max-width: 311px;
+  justify-content: left;
+  gap: 19px;
+  width: 100%;
+  padding: 0;
+  margin-left: 30px;
+}
+
+.pop-up-btn1 {
+  width: 150px;
+  height: 50px;
+  color: white;
+  border: #ff6b00;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 200;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  justify-self: center;
+}
+
+.pop-up-btn2 {
+  width: 150px;
+  height: 50px;
+  color: white;
+  border: #ff6b00;
+  background-color: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 200;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  justify-self: center;
+}
+
+.pop-up-btn1::after {
+  content: url(./svg/Icon-see\ live.svg);
+  padding-left: 10px;
+  display: inline-block;
+}
+
+.pop-up-btn2::after {
+  content: url(./png/git.png);
+  padding-left: 10px;
+  display: inline-block;
+}
+
+.A {
+  display: flex;
+  justify-content: center;
+}
+
```

Comment:
- [ ] As I mentioned in my above comment related to the popUp, even your desktop design for PopUp has some issues. Please fix the UI issues and try to be consistent with the design because it will make your portfolio looks more professional and outstanding.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-21 17-01-21](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/829f0ac2-64f2-424e-81d5-6a938783bf68)

</td>
<td>

![image](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/17e8c915-f5aa-4700-9d80-c2eeb2c48e6d)

</td>
</tr>
</table>

Reply:
- [ ] As requested by previous reviewers, The 2 buttons at the bottom are not visible on mobile version. These buttons are part of the project requirement. Instead of testing the project on your local machine, please test on an online platform like githack to be sure the project displays as expected 👍🏾